### ISSUE TITLE
Two PCs with windows 10 professional failed to start after upgrade docker to latest version

### ISSUE THREAD

**Issue by MinweiWang** [docker/for-win#2274]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID:F6352F84-EE38-4924-8433-1777CBA3E45F/20180726121216

### Expected behavior
After upgrading to latest docker version， docker should start。
### Actual behavior
Docker promoted error，and can't start. Even I did switch to windows containers and reboot windows.
### Information
I have two PCs with windows 10 and both of them reported the errors. I installed ubuntu from Microsoft market store. I'm not sure it is related to this issue.
I booted my PC on July 25th. It prompted to upgrade docker to new version with kubernates supported. I clicked upgrade. After upgrade, the docker prompted error. I do the steps to recover it as follows:
1. rebooted PC, the issue was still there;
2. switch to windows containers, it shows "it will switch to windows containers". But it last long time, and looked it didn't switch to windows containers;
3. I uninstalled ubuntu in windows and rebooted. it didn't work.

**Comment by ebriney** [comment]
Did it work in previous version?
From the diags, I see that you have VMWare installed, it is not compatible with HyperV, can you remove it.
HyperV seems to fail settings an IP on the vm switch but the error message is localized so I don't know what is it.

**Comment by MinweiWang** [comment]
It has been resolved with suggestion on issue #2263.
After upgrade, DockerNAT network adapter was lost on one PC. Broken_dockerNAT network adapter was showed on another PC. which caused Docker can't be started. 
I removed the Borken_dockerNAT and disabled hyper-v, and then rebooted PC and docker prompted to enable hyper-v. After enable hyper-v and docker create correct DockerNAT and it ran.
I'm not sure why DockerNAT is missed or broken, but it is the workaround way to make docker work. 
Thanks for your help!
